test/unit/buybutton.js

```diff
@@ -1,17 +1,16 @@
 import ShopifyBuy from '../../src/buybutton';
 import UI from '../../src/ui';
 
-describe.skip('ShopifyBuy.UI', () => {
-  const configAttrs = {
+describe('ShopifyBuy.UI', () => {
+  const config = new Config({
     domain: 'embeds.myshopify.com',
-    apiKey: 123,
-    appId: 6
-  };
+    storefrontAccessToken: 'meow',
```

Comment:
maybe use a value that's more clearly an access token. Like `access-token-12345` or something?